website/www/site/content/en/documentation/transforms/python/aggregation/combinevalues.md

```diff
@@ -0,0 +1,230 @@
+---
+title: "CombineValues"
+---
+<!--
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# CombineValues
+
+{{< localstorage language language-py >}}
+
+{{< button-pydoc path="apache_beam.transforms.core" class="CombineValues" >}}
+
+Combines an iterable of values in a keyed collection of elements.
+
+See more information in the [Beam Programming Guide](/documentation/programming-guide/#combine).
+
+## Examples
+
+In the following examples, we create a pipeline with a `PCollection` of produce.
+Then, we apply `CombineValues` in multiple ways to combine the keyed values in the `PCollection`.
+
+`CombineValues` accepts a function that takes an `iterable` of elements as an input, and combines them to return a single element.
+`CombineValues` expects a keyed `PCollection` of elements, where the value is an iterable of elements to be combined.
+
```

Comment:
Can you add a sentence here that briefly explains saturation arithmetic?

Reply:
Sure, I added an explanation in the second example that is where the saturated sum is introduced.